### PR TITLE
Add more customization to ticket

### LINKED ISSUE #3: Generic changes to look of RT ticket
* body: nicely formatted view of whole incident (similar to frontend look)
* links - refers to: link back to incident (main url can be found in settings), details_url

### LINKED ISSUE #5: Use custom fields in RT
It should possible to add a mapping to the settings that specifies custom fields and which part of the serialized incident to use for it, which will then be used when creating a new ticket. 

Things that the Sikt Service Center needs:
* kategori: "incident"
* tidspunkt start: incident start
* tidspunkt stop: incident end (if exists)
* kundetjenester: tag kundetjenester
* utstur: tag host
* description: description

### PR DESCRIPTION
**Pull request by johannaengland**
For now we added custom fields, both ones where it is always the same value and where it is a mapping from the incident. 
Links to the Argus incident and the details url where added as well.

Will close #3 and will close #5.

### CODE REVIEW

File: src/argus_ticket_rt.py [hunk 1]
```diff
@@ -25,15 +25,12 @@ def import_settings(cls):
             LOG.exception("Could not import settings for ticket plugin.")
             raise TicketPluginException(f"Request Tracker: {e}")
 
-        if (
-            "username" not in authentication.keys()
-            or "password" not in authentication.keys()
-        ):
+        if "token" not in authentication.keys():
```
Comment (hmpf) [inline src/argus_ticket_rt.py]: You no longer support username/password at all? Why?

File: src/argus_ticket_rt.py [hunk 2]
```diff
@@ -25,15 +25,12 @@ def import_settings(cls):
...
             LOG.error(
-                "Request Tracker: No username or password can be found in the authentication information. Please update the setting 'TICKET_AUTHENTICATION_SECRET'."
+                "Request Tracker: No token can be found in the authentication information. Please update the setting 'TICKET_AUTHENTICATION_SECRET'."
             )
             raise TicketPluginException(
-                "Request Tracker: No username or password can be found in the authentication information. Please update the setting 'TICKET_AUTHENTICATION_SECRET'."
+                "Request Tracker: No token can be found in the authentication information. Please update the setting 'TICKET_AUTHENTICATION_SECRET'."
```
Comment (hmpf) [inline src/argus_ticket_rt.py]: Put the error-message in a variable

File: src/argus_ticket_rt.py [hunk 3]
```diff
@@ -46,14 +43,37 @@ def import_settings(cls):
...
+        )
+        custom_fields = ticket_information.get("custom_fields", {})
+        custom_fields_mapping = ticket_information.get("custom_fields_mapping", {})
+
+        for key, field in custom_fields_mapping.items():
+            if type(field) is dict:
+                # Information can be found in tags
+                custom_fields[key] = incident_tags[field["tag"]]
+            else:
+                if serialized_incident[field] == "infinity":
```
Comment (hmpf) [inline src/argus_ticket_rt.py:69]: Add a comment why "infinity" is magic

File: src/argus_ticket_rt.py [hunk 4]
```diff
@@ -46,14 +43,37 @@ def import_settings(cls):
...
     def create_client(endpoint, authentication):
         """Creates and returns a RT client"""
         try:
-            client = Client(
-                username=authentication["username"],
-                password=authentication["password"],
-                endpoint=endpoint,
+            client = Rt(
+                url=urljoin(endpoint, "REST/2.0"),
+                token=authentication["token"],
```
Comment (hmpf) [inline src/argus_ticket_rt.py]: If the library supports both tokens and username/pasword, why are you limiting it to just token?

**Review by hmpf — CHANGES_REQUESTED** [review]

File: src/argus_ticket_rt.py (hunk 1 above)
Comment (johannaengland) [reply src/argus_ticket_rt.py]: That was an oversight, I changed it accordingly in `create_client`, but accidentally forgot to change it here as well.

File: src/argus_ticket_rt.py (hunk 3 above)
Comment (hmpf) [reply src/argus_ticket_rt.py:69]: So it's the argus infinity and not something RT-specific? Ok.

**Review by hmpf — APPROVED** [review]

**Pull request merged**